### DISCUSSION TITLE
Weighting Part of Training Set More Heavily

### DISCUSSION THREAD

**Discussion by python-4** [deepmodeling/deepmd-kit#124] [Q&A] [answered]
Hi,

I am training a force field with DeepMD and I am thinking if I can weight part of my training set more heavily since they are more accurate than the rest. The first thing come to my mind is to copy the precise data many times in training set. Another idea is to modify the loss function and give weights to all configurations.

Any idea on how to weight part of training set more heavily? Thank you very much!

**Comment by njzjz** [comment, marked as answer]
The newest version has already supported this feature. For example,
```json
"systems": ["data.old0", "data.old1", "data.old2", "data.new0", "data.new1"],
"auto_prob_style": "prob_sys_size; 0:3:1; 3:5:9",
```
makes the new data is 9 times more likely to be trained.
